code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -84,8 +84,7 @@
 	if(istype(W, /obj/item/weapon/rcs) && !src.opened)
 		var/obj/item/weapon/rcs/E = W
 		if(E.rcell && (E.rcell.charge >= E.chargecost))
-			// TODO: Tie into space manager
-			if(!(src.z in config.player_levels))
+			if(!is_level_reachable(src.z)) // This is inconsistent with the closet sending code
 				to_chat(user, "<span class='warning'>The rapid-crate-sender can't locate any telepads!</span>")

```

Comment:
Kind of. There was a config variable called `contact_levels`, which consisted of the asteroid and the main station. That variable has become the `is_station_contact` check we now use. The other is `is_level_reachable`, which as you state, are the levels reachable by space transitions.